apps/src/netsim/NetSimTable.js

```diff
@@ -119,6 +148,38 @@ var NetSimTable = module.exports = function (channel, shardID, tableName, option
   this.lastRefreshTime_ = 0;
 
   /**
+   * Minimum time (in ms) to wait after an invalidation event before attempting
+   * to trigger a refresh request.  This produces a window in which clustered
+   * invalidations can be captured and coalesced together.
+   * @private {number}
+   */
+  this.minimumDelayBeforeRefresh_ = DEFAULT_MINIMUM_DELAY_BEFORE_REFRESH_MS;
+  if (options && options.minimumDelayBeforeRefresh !== undefined) {
```

Comment:
`undefined` is the value if the caller didn't set `minimumDelayBeforeRefresh` at all.  If this comes in `null` then the caller explicitly passed `minimumDelayBeforeRefresh: null` and we should probably throw `TypeError`.  I'll add a check for that.